packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.routing.js

```diff
@@ -12,6 +12,11 @@ export default /* @ngInject */ ($stateProvider) => {
       url: '?ip&dateTime',
       component: 'traffic',
       resolve: {
+        ip: /* @ngInject */ ($transition$) => $transition$.params().ip,
+        dateTime: /* @ngInject */ ($transition$) =>
+          $transition$.params().dateTime
+            ? new Date($transition$.params().dateTime)
+            : null,
```

Comment:
You need to add in the binding in https://github.com/ovh/manager/blob/19a51415c989b8179f230bde4adf0dc648f11632/packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.component.js